CHANGELOG.md

```diff
@@ -6,16 +6,17 @@ Since v0.10.0 this project adheres to [Semantic Versioning](http://semver.org/)
 
 #### Added
 
-* Added more debug logging of MediaWiki requests and responses ([#101])
+* Added more debug logging of MediaWiki requests and responses ([#101], [#106])
 * New GOVERNANCE.md file to explicitly codify the project management principles and provide guidelines for maintenance tasks ([#83], [#105])
 
 #### Changed
 
-* Modernize token handling for login and data-modifying actions ([#100])
+* Modernize token handling for login and data-modifying actions ([#100], [#106])
 
 #### Fixed
 
 * Prevented PHP notice in `WikiFile::getInfo()` for moved or deleted file ([#85])
+* Fixed capitalization of a built-in PHP class ([#106])
```

Comment:
It pertains only to a comment, not code, so perhaps the changelog entry can indicate that?

Reply:
Makes sense, will adjust.